src/en/guide/plugins/creatingAndInstallingPlugins.adoc

```diff
@@ -90,11 +90,20 @@ Adds Quartz job scheduling features
 
 ==== Plugin Configuration
 
+A Grails plugin can be configured in one of the following files:
```

Comment:
Can we be more specific: A Grails Plugin can expose its configuration options to the consuming applications in the following files: